packages/cli/lib/services/test.service.unit.cli-test.ts

```diff
@@ -0,0 +1,374 @@
+import { describe, expect, it } from 'vitest';
```

Comment:
How feasible would it be to add a test for what the service actually generates? I didn't find one in the codebase.

Reply:
Added with https://github.com/NangoHQ/nango/pull/5126/commits/625d0588b9fdc8e289c17f77a296044ea8509c69